GoogleUtilities.podspec

```diff
@@ -34,6 +34,8 @@ other Google CocoaPods. They're not intended for direct public usage.
     es.source_files = 'GoogleUtilities/Environment/**/*.[mh]'
     es.public_header_files = 'GoogleUtilities/Environment/**/*.h'
     es.private_header_files = 'GoogleUtilities/Environment/**/*.h'
+
+    es.dependency 'PromisesObjC', '~> 1.2'
```

Comment:
Separate from this review - but are we comfortable allowing minor version updates of Promises. It might be safer to lock to patch version updates ...